iree/compiler/Codegen/LLVMGPU/LLVMGPUPipelining.cpp

```diff
@@ -45,6 +45,7 @@ static void getPipelineStages(
     if (op.hasAttr(kPipeliningGlobalLoad))
       addDepOps(loadDep, &op, forOp.getBody());
   }
+  // Create a schedule with stage 1 ops first followed by stage 0 ops.
```

Comment:
Nit: Something more descriptive would be nice (like what you told me in our 1:1, w.r.t modulo scheduling)